leakcanary-android-core/src/main/java/leakcanary/internal/HeapAnalyzerService.kt

```diff
@@ -65,7 +65,7 @@ internal class HeapAnalyzerService : ForegroundService(
     val fullHeapAnalysis = when (heapAnalysis) {
       is HeapAnalysisSuccess -> heapAnalysis.copy(
         dumpDurationMillis = heapDumpDurationMillis,
-        metadata = heapAnalysis.metadata + ("Heap dump reason" to heapDumpReason)
+        metadata = heapAnalysis.metadata + ("Heap dump reason" to heapDumpReason!!)
```

Comment:
`!!` should move where `heapDumpReason` is declared.

Reply:
done.